src/main/resources/Quests.json

```diff
@@ -87,11 +90,13 @@
                   "itemQuantity":1
                }
             ],
+            "chunkLocation":null,
             "extraInfo":""
          },
          {
             "stepNumber":5,
             "stepDescription":"Speak to Cook",
+            "itemReqs":null,
```

Comment:
@apan145 will this work okay for importing, or should this be handled differently?

Reply:
The JSON is parsed just fine, but if we let itemReqs be set to null, we have to check for it. I think it is easier codewise to set itemReqs to an empty array if the step requires no items.

Reply:
```suggestion
            "itemReqs":[],
```

Reply:
There's a bunch of those I gotta change then, gimme a few